uk_bin_collection/uk_bin_collection/councils/CeredigionCountyCouncil.py

```
@@ -82,7 +82,32 @@ def parse_data(self, page: str, **kwargs) -> dict:
                 )
             )
 
-            address_dropdown.select_by_visible_text(house_number)
+            # Fuzzy match: try exact first, then startswith, then contains
+            matched = False
+            try:
+                address_dropdown.select_by_visible_text(house_number)
+                matched = True
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)